python/lsst/daf/butler/registry/queries/expressions/parser/exprTree.py

```diff
@@ -240,11 +245,32 @@ def __str__(self) -> str:
         return "{value}".format(**vars(self))
 
 
+class UuidLiteral(LiteralNode):
+    """Node representing a UUID literal.
+
+    Parameters
+    ----------
+    value : `UUID`
+        Literal value.
+    """
+
+    def __init__(self, value: UUID):
+        Node.__init__(self)
```

Comment:
```suggestion
        super().__init__()
```
?  Or maybe `LiteralNode.__init__(self)`?  Or is using an indirect base-class `__init__` intentional?

Reply:
That was copy-paste, need to use `super()` indeed.